pages/sponsorship.tsx

```diff
@@ -55,8 +55,8 @@ export default withPageMetadata((props: WithPageMetadataProps) => (
     </p>
 
     <p>
-      In 2017 DDD Perth had 330 attendees from 360 sold tickets; this makes us the biggest software conference in
-      Western Australia.
+      In 2018 DDD Perth had 470 attendees from 510 sold tickets; this makes us the biggest software conference in
```

Comment:
Thoughts on moving these numbers into ./config/2018.ts or ./config/conference.ts? In terms of historical data the former seems good?

Reply:
I know these numbers are referenced elsewhere on the site too, so makes sense to put in config. I'll have a look at changing.